scss/_themes.scss

```diff
@@ -5,8 +5,8 @@
 
 /// scss-lint:disable SpaceAfterComma
 
-@include oColorsSetColor('o-buttons-teal-3', #17484C);
-@include oColorsSetColor('o-buttons-teal-4', #0A2F33);
+@include oColorsSetColor('o-buttons-teal-3', #17484c);
```

Comment:
This is weird right? why are we creating random colours here in the first place. I'll look into this (separate to this PR)

I mean, at the very least there should be a comment as to why we did this.